### PR TITLE
JAMES-2544 Simplify testing with RabbitMQ

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/DockerRabbitMQ.java [hunk 1]
```diff
@@ -164,4 +168,24 @@ public void reset() throws Exception {
...
+        return new URIBuilder()
+                .setScheme("amqp")
+                .setHost(getHostIp())
+                .setPort(getPort())
+                .build();
+    }
+
+    public URI managementUri() throws URISyntaxException {
+        return new URIBuilder()
+                .setScheme("amqp")
```
Comment (mbaechler) [inline backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/DockerRabbitMQ.java]: http

Comment (aduprat) [reply backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/DockerRabbitMQ.java]: Right !

**Comment by aduprat** [comment]
```
[ERROR] Errors: 
[ERROR]   VacationIntegrationTest.jmapVacationShouldNotSendNotificationTwice:237 » ConditionTimeout
```
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**